libobs/obs-internal.h

```diff
@@ -17,6 +17,8 @@
 
 #pragma once
 
+#include <stdatomic.h>
```

Comment:
I see that other code in this file is using volatile even though atomic is more appropriate. If there is an underlying reason let me know and we can use volatile here, too.